recipes-bsp/u-boot/files/0028-iot2050-Add-support-for-configuring-M.2-connector.patch

```diff
@@ -0,0 +1,534 @@
+From 413f9db7beba137c6ea5fcdd81f5e2c7c955cdba Mon Sep 17 00:00:00 2001
+From: Jan Kiszka <jan.kiszka@siemens.com>
+Date: Tue, 11 Oct 2022 13:23:21 +0200
+Subject: [PATCH 28/32] iot2050: Add support for configuring M.2 connector
+
+The M.2 slots of the related IOT2050 variant need to be configured
+according to the plugged cards. This tries to detect the card using the
+M.2 configuration pins of the B-key slot. If that fails, a U-Boot
+environment variable can be set to configure manually. This variable is
+write-permitted also in secure boot mode as it is not able to undermine
+the integrity of the booted system.
```

Comment:
manual configuration prior to auto-detection.
so it's not "If that fails, a U-Boot environment variable can be set to configure manually"

whether or not fails, manual configuration should be first priority if existed

Reply:
It is meant as "if auto-configuration does not work for you, you can set up manual configuration". I will try to reword this to make it clearer.

Reply:
"For the case auto configuration does not work reliably, it is possible to set
the U-Boot environment variable "m2_manual_config" to select the mode manually:"